### PR TITLE
fixed config

### PR DESCRIPTION
**Pull request by GabiRP**
Fixed Scp096SeenMessage, and Scp096NewTargetMessage

### CODE REVIEW

File: Scp096Notifications/Plugin.cs [hunk 1]
```diff
@@ -18,14 +18,16 @@ public class Scp096Notifications : Plugin<Config>
...
+        public static Plugin Singleton;
         private static EventHandlers Handler;
 
         public override void OnEnabled()
         {
+
             if (!Config.IsEnabled) return;
-            Singleton = new Scp096Notifications();
-            Handler = new EventHandlers();
+            Log.Info($"{Name} plugin has been enabled" + Author);
```
Comment (Thundermaker300) [inline Scp096Notifications/Plugin.cs:28]: This log is redundant, `base.OnEnabled()` already logs a message.

**Review by Thundermaker300 — CHANGES_REQUESTED** [review]

File: Scp096Notifications/Plugin.cs (hunk 1 above)
Comment (Thundermaker300) [inline Scp096Notifications/Plugin.cs]: Redundant log, base.OnEnabled() does this.

Comment (Thundermaker300) [reply Scp096Notifications/Plugin.cs]: (Not sure why it's selecting how it is, I'm just referring to the Log.Info line)

Comment (GabiRP) [reply Scp096Notifications/Plugin.cs]: oh, its true, wait

Comment (GabiRP) [reply Scp096Notifications/Plugin.cs]: Ok, I just deleted that line

**Pull request merged**